rotkehlchen/tests/api/blockchain/test_substrate.py

```diff
@@ -80,7 +80,7 @@ def test_add_ksm_blockchain_account(rotkehlchen_api_server):
     result = assert_proper_response_with_result(response)
 
     # Check per account
-    account_balances = result['per_account']['KSM'][SUBSTRATE_ACC1_KSM_ADDR]
+    account_balances = result['per_account']['ksm'][SUBSTRATE_ACC1_KSM_ADDR]
```

Comment:
same comments here and elsewhere for `account_balances['assets']['KSM']`.

Will stop repeating it further in the PR